packages/python-packages/apistubgentest/setup.py

```diff
@@ -63,5 +63,12 @@
     ],
     zip_safe=False,
     packages=find_packages(),
+    extras_require={
+        ":python_version<'3.7'": ["qsharp>=1.0.33,<2.0"],
+        "cirq": [
+            "cirq-core>=1.3.0,<=1.4.1",
+            "cirq-ionq>=1.3.0,<=1.4.1",
+        ]
+    },
```

Comment:
Does this mean every Azure package that has an optional dependency will need it added here?

Reply:
This is for testing.